src/status_im2/contexts/quo_preview/share/qr_code.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im2.contexts.quo-preview.share.qr-code
```

Comment:
Please add all the styles in this ns inside the `:style` key :)

Reply:
@ulisesmac this comes up every now and then. We haven't added any styles for quo preview screens to this structure. 
There is also some discussion that we will eventually migrate all of these preview screens to a more solid application, a la story book. 

Do you prefer I start this process and create view and style views for the preview screen?

Reply:
Oh no, it's ok, I know we don't create a separate style file.

I meant, add styles inside the `:style` key:

```clojure
[component {:style  {:flex 1} ;; <- here
            :prop-1 1
            :prop-2 2}]
;; Instead of:
[component {:flex 1 
            :prop-1 1
            :prop-2 2}]
```

for the whole file, because there're many components not following this pattern

Reply:
ah apologies! I misread, your explanation was clear. :)